packages/marko-web/express/error-handlers.js

```diff
@@ -47,7 +48,13 @@ const redirectOrError = ({
     } else {
       render(res, { statusCode, err, template });
     }
-  }).catch(() => render(res, { statusCode, err, template }));
+  }).catch(async () => {
+    try {
+      render(res, { statusCode, err, template });
+    } catch (e) {
+      error('Unable to render error template for', err);
```

Comment:
These errors should also be logged, not just printed to the console :)

Reply:
Done; an `onFatalError` handler can now be passed to `start-server` to do _whatever_. This will fall back to the `onAsyncBlockError` handler for backwards compatibility, but will allow it to be customized by passing the new argument.